chain/messagepool/messagepool.go

```diff
@@ -139,9 +139,14 @@ type MessagePool struct {
 
 	keyCache *lru.Cache[address.Address, address.Address]
 
-	curTsLk sync.RWMutex // DO NOT LOCK INSIDE lk
+	stateLk sync.RWMutex
 	curTs   *types.TipSet
 
+	// Use this lock if you need to transact between the TS and the Message pool in a consistent manner
+	//all changes to curTs must be wrapped in a transactionLk
+	//You do not need this lock and only need curTsLk if you want a consistent view of the state without changing the state
```

Comment:
Change name here from curTsLk to stateLk in accordance with the change on line 142

Reply:
ty!!! good catch

Reply:
#44bbe7fdc

Reply:
#44bbe7fdc4271dc29218415a9a0c89d2417bab2f

Reply:
https://github.com/filecoin-project/lotus/commit/44bbe7fdc4271dc29218415a9a0c89d2417bab2f